package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
-    "start": "npm run build-themes && ng serve --aot --sourcemaps=false",
```

Comment:
`--aot`

Want to add another command like `start-jit` that is set up that way?